seqr/views/apis/staff_api_tests.py

```diff
@@ -375,6 +375,45 @@
     'entity:discovery_id': u'HG00731', 'Chrom-1': u'1',
     'inheritance_description-1': 'de novo'}
 
+EXPECTED_SAMPLE_METADATA_ROW = {
+    "MME": "N",
+    "affected_status": "Affected",
+    "ancestry": "",
+    "ancestry_detail": "",
+    "data_type": "WES",
+    "date_data_generation": "2017-02-05",
+    "dbgap_submission": "No",
```

Comment:
You need to choose a sample row with airtable data and gene level data, ideally a gene thats NOT "Known gene for phenotype"

Reply:
I'm sure about "Known gene for phenotype" gene.